### ISSUE TITLE
Dynamips identifier is already used by another router

### ISSUE THREAD

**Issue by noplay** [GNS3/gns3-server#327]
https://community.gns3.com/thread/22346

**Comment by ehlers** [comment]
GNS3 version 1.4.0.dev9 on Darwin (64-bit) with Python 3.5.0 Qt 5.5.0.

Simple to reproduce:

Open a project (don't start it), which has at least one dynamics router, e.g.
![screenshot](https://cloud.githubusercontent.com/assets/896406/10268055/0d78cc0e-6aad-11e5-8740-26b95e8de715.png)

Then just open it again, the console shows:
```
Server error from http://192.168.1.10:8000: R1: Dynamips identifier 2 is already used by another router
Server error from http://192.168.1.10:8000: ESW1: Dynamips identifier 1 is already used by another router
```

**Comment by grossmj** [comment]
The error is likely to be in 1.3.x as well.

**Comment by noplay** [comment]
Confirmed with 1.3.11 and 1.4.0dev9

**Comment by noplay** [comment]
Not present in 1.3.10

**Comment by noplay** [comment]
I have moved the code managing the attribution of dynamips_id to the manager. This allow to proper clean the list when the project is closed. Also I add tests for this specific part.

**Comment by michaelkasede** [comment]
Hi noplay, thanx for the fix. How do I apply the fix to GNS3 remote server on ubuntu 16.04?
